src/types/common.ts

```diff
@@ -17,6 +17,16 @@ export enum Status {
   Error,
 }
 
+export enum Errors {
```

Comment:
```suggestion
export enum ErrorCode {
```
nit to make it more explicit